klpbuild/config.py

```diff
@@ -29,13 +30,19 @@
 
 class Config:
     def __init__(self, lp_name, lp_filter, kdir=False, data_dir=None, skips="", working_cs={}):
-        work_dir = os.getenv("KLP_WORK_DIR")
-        if not work_dir:
-            raise ValueError("KLP_WORK_DIR should be defined")
+        logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+        home = os.getenv("HOME")
+        self.user_conf_file = Path(home, ".config/klp-build/config")
+        if not self.user_conf_file.is_file():
+            logging.warning(f"Warning: user configuration file not found")
+            # If there's no configuration file assume fresh install.
+            # Prepare the system with a default environment and conf.
+            self.setup_user_env(Path(home, "klp-build"))
```

Comment:
Here I think that we can rename the directory to ``klp``, since klp-build can be confused with the project name. We were using ``kgr`` for historic reasons, but currently we use ``klp`` everywhere.